document/core/valid/types.rst

```diff
@@ -240,7 +240,7 @@ Import Subtyping
 
 When :ref:`instantiating <exec-module>` a module,
 :ref:`external values <syntax-externval>` must be provided whose :ref:`types <valid-externval>` are *matched* against the respective :ref:`external types <syntax-externtype>` classifying each import.
-In some cases, this allows for a simple form of subtyping, as defined here.
+In some cases, this allows for a simple form of subtyping (written ":math:`\matchesexterntype`" formally), as defined here.
```

Comment:
```suggestion
In some cases, this allows for a simple form of subtyping (written ":math:`\matches`" formally), as defined here.
```

Reply:
Sorry, scratch that, that only works on a later repo.

Reply:
Yeah I tried it and it doesn’t work, which is why I went for this one 😅